Sources/App/Features/Tickets/Models/TitoTicket.swift

```diff
@@ -3,6 +3,7 @@ import Foundation
 struct TitoTicket: Codable {
     struct Metadata: Codable {
         let canBookDropInSession: Bool?
+        let canViewDropInSessionParticipants: Bool?
```

Comment:
This is the JSON you can add to ticket types. Currently added to Internal but add it to as many types as you want (so talk show hosts). Scalable and configurable for future.